client/app/tasks/types/taskFormType.ts

```diff
@@ -0,0 +1,6 @@
+export type TaskFormType = {
```

Comment:
Question! Is there any reason you can not use `taskSchema`? Also I think it's better to defined `TaskSchema` because it is type

Reply:
fixed it
https://github.com/HAK2024/fairy-share/pull/80/commits/be08863833c005d9e042264e03364efc6b726094
https://github.com/HAK2024/fairy-share/pull/80/commits/b0163e99346d6abbad8adf007223ea284e9dc585